ui/src/app/views/confirm-delete-modal/component.css

```diff
@@ -0,0 +1,23 @@
+:host::ng-deep .modal-dialog {
+  width: 18rem;
+}
+
+:host::ng-deep .modal-content {
+  border-radius: 8px;
+}
+
+.modal-title-text {
+  margin-top: 0;
+  font-size: 20px;
+  line-height: 28px;
+  color: #302973;
+  font-weight: 500;
+}
+
+.modal-title-subtext {
+  margin-top: 0.2rem;
+  font-size: 14px;
+  color: #4A4A4A;
+  font-weight: 300;
+  line-height: 28px;
+}
```

Comment:
You probably want to configure your editor so you don't introduce this noise (absence of trailing newline) on PRs.